addons/payment/static/src/js/payment_form_mixin.js

```diff
@@ -12,14 +12,14 @@ odoo.define('payment.payment_form_mixin', require => {
          * @override
          */
         start: async function () {
+            this.txContext = {};
```

Comment:
To give a chance for the line to stay where it is.
```suggestion
            this.txContext = {};  // Synchronously initialize txContext before any await.
```